packages/legend-query/style/_query-builder.scss

```diff
@@ -35,6 +35,51 @@
     background: var(--color-dark-grey-50);
   }
 
+  &__sub-header {
+    @include flexVCenter;
+
+    height: 2.8rem;
+    background: darkgray;
```

Comment:
Please don't use color that we don't define here. Also, we could make use of darker grey that we already have.